pom.xml

```diff
@@ -489,6 +489,7 @@
 				<groupId>org.codehaus.mojo</groupId>
 				<artifactId>versions-maven-plugin</artifactId>
 				<configuration>
+					<excludeProperties>changelist</excludeProperties>
```

Comment:
Sorry, i dont really understand why this is needed. I also read the commit message.

Reply:
`make update-dependencies` doesn't just update lines like `<maven-assembly-plugin.version>3.7.0</maven-assembly-plugin.version>`, but it also changes the `<changelist>-SNAPSHOT</changelist>` to `<changelist>1.0.4</changelist>`, presumably because the `-SNAPSHOT` looks like a version and is referenced by our `parent` artifact, and for that the latest available version is inserted.
We want the `-SNAPSHOT` to remain there in order to produce snapshots on default builds; our CI/CD-build on main explicitly disables snapshots. Therefore, exclude the `changelist` property from updates.